traits/memberships/src/impl_nonfungibles.rs

```diff
@@ -1,13 +1,16 @@
 use crate::*;
 use frame_support::{
-    pallet_prelude::DispatchError, sp_runtime::traits::Zero,
+    pallet_prelude::DispatchError,
+    sp_runtime::{str_array, traits::Zero},
     traits::tokens::nonfungibles_v2 as nonfungibles,
 };
 
 const ATTR_MEMBER_TOTAL: &[u8] = b"membership_member_total";
 const ATTR_MEMBER_RANK: &[u8] = b"membership_member_rank";
 const ATTR_MEMBER_RANK_TOTAL: &[u8] = b"membership_rank_total";
 
+pub const ASSIGNED_MEMBERSHIPS_ACCOUNT: [u8; 32] = str_array("modlkv/mbrsh_reserved");
```

Comment:
```suggestion
pub const ASSIGNED_MEMBERSHIPS_ACCOUNT: [u8; 32] = str_array("memberships/assigned_memberships");
```
`mod` is for pallets and `kv` kreivo and this implementation is neither :)